src/Build/BuildCheck/Infrastructure/TracingReporter.cs

```diff
@@ -13,6 +13,11 @@ internal class TracingReporter
 {
     internal Dictionary<string, TimeSpan> TracingStats { get; } = new();
 
+    // Infrastructure time keepers, examples for now
+    internal TimeSpan analyzerAcquisitionTime;
+    internal TimeSpan analyzerSetDataSourceTime;
+    internal TimeSpan newProjectAnalyzersTime;
+
```

Comment:
Please comment more in detail of what is the purpose of those - or remove. So that this doesn't stay in code forewer unused

Reply:
I readded the timers to their respective functions. I think it might have gotten lost during the merge.